### DISCUSSION TITLE
Can I use Huggingface models to init T5 or mT5?

### DISCUSSION THREAD

**Discussion by puraminy** [google-research/text-to-text-transfer-transformer#832] [Q&A]
For example I want to use an mT5 model fine-tuned on translation with huggingface models:

https://huggingface.co/persiannlp/mt5-base-parsinlu-translation_en_fa

Can I use it as start point for an mT5 model developed by your team? If yes, how?

**Comment by craffel** [comment]
Do you mean using a checkpoint saved by HF using our training code (mesh TF etc)? No, this is not possible. But, you can use our data pipelines with HF, see e.g. https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/models/hf_model.py
